source/guide_focalboard.rst

```diff
@@ -0,0 +1,119 @@
+.. highlight:: console
+
+.. author:: Thomas Kammerer <https://kammerer.tk/>
+
+.. tag:: web
+.. tag:: project-management
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/focalboard.svg
+      :align: center
+
+##########
+Focalboard
+##########
+
+.. tag_list::
+
+Focalboard_ is an open source, self-hosted alternative to Trello, Notion, and Asana. 
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+It's recommended to use a PostgreSQL database. Install and configure PostgreSQL server as described in the :lab:`PostgreSQL guide <guide_postgresql>`.
+
+Installation
+============
+
+Download the latest release_ to your home directory and extract the application.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ wget https://github.com/mattermost/focalboard/releases/download/v0.6.1/focalboard-linux.tar.gz
+ [isabell@stardust ~]$ tar --extract --gzip --file focalboard-linux.tar.gz
+ [isabell@stardust ~]$ rm focalboard-linux.tar.gz
+ [isabell@stardust ~]$
+
+
+Configuration
+=============
+
+You need to modify ``~/focalboard-app/config.json``:
```

Comment:
The formatting here [is broken](https://deploy-preview-966--uberlab.netlify.app/guide_focalboard.html#configuration), but I am not quite sure why. Adding a blank line between this one and the actual list fixes it.